### PR TITLE
Primary Navigation; Update to Latest Design

### PR DESCRIPTION
**Pull request by SudharakaP**
This updates the primary navation in accordance with the latest design documentation.

Part of https://github.com/opencollective/opencollective/issues/3009

### Screenshots

<details>
<summary>Mobile</summary>

Screen Shot 2020-07-28 at 16 58 01
</details>

<details>
<summary>Desktop</summary>

image
</details>

### CODE REVIEW

**Comment by SudharakaP** [comment]
@Betree : I've done the prop change and the order of the avatar/username. For the other mobile menu thing please feel free to create a PR and assign it to me. :smile:

File: components/SearchForm.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import StyledInput from './StyledInput';
 
 const SearchInputContainer = styled(Flex)`
   border: solid 1px var(--silver-four);
-  border-radius: 20px;
+  border-radius: 6px;
```
[the remark this answers is not in the document]
Comment (SudharakaP) [reply components/SearchForm.js]: @Betree : This part is done. :+1:

[the remark this answers is not in the document]
Comment (Betree) [reply components/SearchForm.js]: Are you sure this is resolved? I don't see where we take a prop to customize the border radius. `20px` should be the default, and `6px` should only be set by the TopBar.

**Comment by Betree** [comment]
> I've done the prop change and the order of the avatar/username

@SudharakaP if you check the design there's also a change of font color

File: components/SearchForm.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (SudharakaP) [reply components/SearchForm.js]: @Betree : Ah, I think I somehow got confused. Just added the prop. I think now it should be fine. Let me know. :smile:

**Comment by SudharakaP** [comment]
> > I've done the prop change and the order of the avatar/username
> 
> @SudharakaP if you check the design there's also a change of font color

Just changed the color and font size and font weight according to the figma design. :smile:

**Comment by Betree** [comment]
There's still a small design mismatch with the caret color

image
image

**Comment by SudharakaP** [comment]
> There's still a small design mismatch with the caret color
> 
> image
> image

@Betree : Ah, I've missed that. Done. :+1:

**Comment by SudharakaP** [comment]
> @Betree : I've done the prop change and the order of the avatar/username. For the other mobile menu thing please feel free to create a PR and assign it to me. smile

Sorry this didn't make sense. I think I was sleep deprived a bit. :smile: 

> Good to merge +1 We'll still need to implement the mobile menu, and design is going to make a small update for mobile.

@Betree : Yes I will open another PR for this one. :+1:

**Pull request merged**